src/web/mjs/connectors/Baozimh.mjs

```diff
@@ -0,0 +1,47 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Baozimh extends Connector {
```

Comment:
Is this website not related to the `SinMH` template which is used for other websites?

Reply:
O how can you tell? I don't know how to see that